components/ResultsPageComponents/StaffSuggestions/StaffSuggestionsChart.jsx

```diff
@@ -0,0 +1,259 @@
+    },
+    {
+      key: 4,
+      radiusSW: true,
+      quoteTop: "10px",
+      quoteLeft: "830px",
+      quote: "I am 20 mins from the station so car pool could work well.",
+    },
+    {
+      key: 5,
+      radiusSW: true,
+      quoteTop: "105px",
+      quoteLeft: "855px",
+      quote: "Provide dedicated bike lanes on arterial roads.",
+    },
+    {
+      key: 6,
+      radiusSW: true,
+      quoteTop: "200px",
+      quoteLeft: "875px",
+      quote:
+        "I have 2 toddlers in tow that need to be dropped off at childcare I can't think of any option other than driving.",
+    },
+    {
+      key: "7a",
+      radiusNW: true,
+      quoteTop: "430px",
+      quoteLeft: "818px",
+      quote:
+        "Flexibility to work from home for three days,  maybe one week a month.",
+    },
+    {
+      key: "7b",
+      radiusNW: true,
+      quoteTop: "510px",
+      quoteLeft: "740px",
+      quote: "Move to electric powered vehicles.",
+    },
+  ];
+
+  function QuoteBox(props) {
+    const {
+      radiusNW,
+      radiusNE,
+      radiusSE,
+      radiusSW,
+      quoteTop,
+      quoteLeft,
+      quote,
+    } = props;
+    const borderRadius = "30px";
+    return (
+      <Box
+        position="absolute"
+        top={quoteTop}
+        left={quoteLeft}
+        borderRadius={`${radiusNW ? "0px" : borderRadius} 
+            ${radiusNE ? "0px" : borderRadius} 
+            ${radiusSE ? "0px" : borderRadius} 
+            ${radiusSW ? "0px" : borderRadius}`}
+        width="300px"
+        border="2px dashed #044B7F"
+        px="10px"
+        py="10px"
+        fontSize="13px"
+      >
+        &quot;{quote}&quot;
+      </Box>
+    );
+  }
+
+  function BubbleText(props) {
+    const { marginLeft, fontSize, text } = props;
+    return (
+      <Text ml={marginLeft} fontSize={fontSize}>
+        {text}
+      </Text>
+    );
+  }
+
+  function Bubble(props) {
+    // console.log("props: ", props);
+    const {
+      bubbleTop,
+      bubbleLeft,
+      bubbleBackgroundColour,
+      bubbleDiameter,
+      bubbleTextColour,
+      textMarginLeft,
+      textFontSize,
+      textString,
+      valueMarginLeft,
+      valueFontSize,
+      valueString,
+    } = props;
+    return (
+      <Box position="absolute" top={bubbleTop} left={bubbleLeft}>
+        <Flex
+          justifyContent="center"
+          direction="column"
+          bg={bubbleBackgroundColour}
+          w={bubbleDiameter}
+          h={bubbleDiameter}
+          borderRadius="50%"
+          color={bubbleTextColour}
+        >
+          <BubbleText
+            marginLeft={textMarginLeft}
+            fontSize={textFontSize}
+            text={textString}
+          />
+          <BubbleText
+            marginLeft={valueMarginLeft}
+            fontSize={valueFontSize}
+            text={valueString}
+          />
+        </Flex>
+      </Box>
+    );
+  }
+
+  return (
+    <Box position="relative" mb="400px" b width="1200px" height="3000px">
```

Comment:
There must be some typo, ```height="3000px"``` is causing a huge white space below the img.

<img width="780" alt="image" src="https://user-images.githubusercontent.com/88268603/185565474-cadd9b1e-250c-488c-8d2b-23a64d688e66.png">


1. Should we use % or `vw`/`vh` for width and height? Any limitations on that?
2. Should we use Flexbox instead of box? Or there is a reason for selecting Box instead?

Reply:
Setting the width and height does not make a difference in the presentation of the chart itself. But:

1. I fixed the height to match the height of the chart so that subsequent content will not appear behind the chart, or too far below it.
2. I specified the width to match the layout settings. Again, this does not affect the presentation of the chart per se. Just serves more of a reminder to future devs that there is a width setting to be aware of.
3. Yes, the "height="3000px" was in error. I have changed that.